balance-io/balance-wallet
# i18n internationalization

### PR Checklist (check all)

* [x] Updated `CHANGELOG.md` to describe the included fixes and changes made
* [x] Included issue number on the description of the PR
* [x] Commented on the relevant issue thread about this PR

### Issue number

https://github.com/balance-io/balance-wallet/issues/98

### Changelog

* Add i18n internationalization

### Test procedure

* Make sure to have the following version of `balance-common`: https://github.com/PierrickGT/balance-common/tree/features%2Fwallet_languages
* Use `yarn link` (https://yarnpkg.com/en/docs/cli/link) in order to test this version in local
* Build the new version with `npm run build` in the `balance-common` folder
* If you want to test the French translation, run the XCode simulator and change the language in `Settings > General > Language and Region`.
Restart the Balance app to see the French translation.
